src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.RemoteServer.cs

```diff
@@ -344,16 +335,25 @@ public async Task SendAsync_HttpRequestMsgResponseHeadersRead_StatusCodeOK(Confi
             }
         }
 
-        [OuterLoop("Uses external servers")]
+        [OuterLoop("Uses external servers", TestPlatforms.Any & ~TestPlatforms.Browser)]
         [Theory, MemberData(nameof(RemoteServersMemberData))]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/37669", TestPlatforms.Browser)]
         public async Task PostAsync_CallMethodTwice_StringContent(Configuration.Http.RemoteServer remoteServer)
         {
             using (HttpClient client = CreateHttpClientForRemoteServer(remoteServer))
             {
                 string data = "Test String";
                 var content = new StringContent(data, Encoding.UTF8);
-                content.Headers.ContentMD5 = TestHelper.ComputeMD5Hash(data);
+
+                if(PlatformDetection.IsBrowser)
+                {
+                    // [ActiveIssue("https://github.com/dotnet/runtime/issues/37669", TestPlatforms.Browser)]
+                    content.Headers.Add("Content-MD5-Skip", "browser");
+                }
+                else
```

Comment:
Why is this preferable over not running the test for browser at all?

Reply:
We don't have MD5, but there are other features the test is testing, not just payload integrity.